runtime/pkg/fileutil/fileutil.go

```diff
@@ -125,3 +126,19 @@ func ForceRemoveFiles(paths []string) {
 		_ = os.Remove(path)
 	}
 }
+
+func ExpandHome(path string) (string, error) {
+	if path == "" || path[0] != '~' {
+		return path, nil
+	}
+
+	usr, err := user.Current()
+	if err != nil {
+		return "", err
+	}
+	if usr.HomeDir == "" {
+		return "", fmt.Errorf("home not set to exapand file %s", path)
+	}
```

Comment:
Typo. Maybe clarify like `"cannot expand '~' in path %q because the current user doesn't have a home directory"`